objectivec/GPBDescriptor.m

```diff
@@ -215,10 +216,33 @@ - (void)setupExtensionRanges:(const GPBExtensionRange *)ranges count:(int32_t)co
   extensionRangesCount_ = count;
 }
 
+- (void)setupContainingMessageClassName:(const char *)msgClassName {
```

Comment:
There are a few method here that use the `setup...` pattern, so I was keeping that at the moment.